examples/ruby/ruby_client.rb

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env ruby
+
+# Load the gem
+require 'openapi_client'
+
+CERT_DIR = 'config/https'
+SSL_CERT_FILE = 'ca.crt'
+CONJUR_CERT_FILE = 'conjur.crt'
+CONJUR_KEY_FILE = 'conjur.key'
+
+ACCOUNT="dev"
+LOGIN="admin"
+ADMIN_API_KEY=ENV["CONJUR_ADMIN_API_KEY"]
+
+# Setup client configuration
+config = OpenapiClient.configure
+config.scheme = "https"
+config.host = "localhost"
+# config.debugging = true
+config.username = LOGIN
+config.password = ADMIN_API_KEY
+config.verify_ssl = true
+config.ssl_ca_cert = File.join(CERT_DIR, SSL_CERT_FILE)
+config.cert_file = File.join(CERT_DIR, CONJUR_CERT_FILE)
+config.key_file = File.join(CERT_DIR, CONJUR_KEY_FILE)
+
+# Authenticating admin using basicAuth
+authn_instance = OpenapiClient::AuthnApi.new
+token = nil
+puts "Authenticating admin..."
+begin
+  token = authn_instance.authenticate(account=ACCOUNT, login=LOGIN, body=ADMIN_API_KEY, opts={accept_encoding: 'base64'})
+  puts "Base64 encoded token: #{token}"
+rescue OpenapiClient::ApiError => e
+  puts "Exception when calling AuthnApi->authenticate: #{e}"
+  exit(1)
+end
+
+# Change admin password using basicAuth
+new_password = "N3w-Passw0rd!"
+puts "\nChanging admin password..."
```

Comment:
```suggestion
puts
puts "Changing admin password..."
```

Reply:
You can apply this pattern to other instances where you embed a newline in a `puts`